### PR TITLE
Updated language for request composer launch.

### PR DESCRIPTION
**Pull request by mcohoon**

### CODE REVIEW

File: templates/request-composer/index.html [hunk 1]
```diff
@@ -3,15 +3,28 @@
 {% block content %}
 <section>
   <h2>Overview</h2>
-  <p>This demo illustrates some of the more advanced features of the <a href="{{site.tech.reporting_api_v4.url}}">{{site.tech.reporting_api_v4.name}}</a>. This tool illustrates how to compose various requests including:
+  <p>
+    The purpose of the Request Composer is to illustrate a small subset of the <a href="{{site.tech.reporting_api_v4.url}}">{{site.tech.reporting_api_v4.name}}</a> features.
```
Comment (philipwalton) [inline templates/request-composer/index.html]: Do we want to say "small" here?

File: templates/request-composer/index.html [hunk 2]
```diff
@@ -3,15 +3,28 @@
...
     <li><a href="{{site.tech.reporting_api_v4.url}}/advanced#cohorts">Cohort requests</a></li>
   </ul>
-  <p>To simplify the processing of the API responses, this tool applies some additional limiitations on the requests that can be made. For full specifications of the API see the <a href="{{site.tech.reporting_api_v4.url}}/rest/v4/reports/batchGet">reference documentation</a>.
+  <p>
+    To simplify the processing of the API responses, this tool applies some additional
+    limitations on the requests that can be made. For full specifications of the API
+    see the
+    <a href="{{site.tech.reporting_api_v4.url}}/rest/v4/reports/batchGet">reference documentation</a>.
+    There are other features of the Analytics Reporting API V4, such as 
+    <a href="{{site.tech.reporting_api_v4.url}}/basics#expressions">metric expression</a>,
```
Comment (philipwalton) [inline templates/request-composer/index.html]: s/expression/expressions/

File: templates/request-composer/index.html [hunk 3]
```diff
@@ -3,15 +3,28 @@
...
+    <a href="{{site.tech.reporting_api_v4.url}}/basics#multiple_date_ranges">multiple date ranges</a>, and
+    <a href="{{site.tech.reporting_api_v4.url}}/basics#filtering_1">advanced filter configuration</a>
```
Comment (philipwalton) [inline templates/request-composer/index.html]: Do we also want to mention new segment syntax?

Comment (mcohoon) [inline templates/request-composer/index.html]: done.

File: templates/request-composer/index.html (hunk 2 above)
Comment (mcohoon) [inline templates/request-composer/index.html]: done

File: templates/request-composer/index.html (hunk 1 above)
Comment (mcohoon) [inline templates/request-composer/index.html]: done.

**Comment by philipwalton** [comment]
Hmm, I ran this locally and it doesn't seem to include the changes from #223. Can you rebase/merge against the `request-composer` branch?

**Comment by mcohoon** [comment]
I merged from the upstream/request-composer and it can now be merged automatically.

**Comment by philipwalton** [comment]
Running `gulp-lint` shows one error, otherwise LGTM.

**Pull request merged**